framework/common/vk_common.cpp

```diff
@@ -96,15 +96,19 @@ VkShaderStageFlagBits find_shader_stage(const std::string &ext)
 	}
 	else if (ext == "rgen")
 	{
-		return VK_SHADER_STAGE_RAYGEN_BIT_NV;
+		return VK_SHADER_STAGE_RAYGEN_BIT_KHR;
 	}
 	else if (ext == "rmiss")
 	{
-		return VK_SHADER_STAGE_MISS_BIT_NV;
+		return VK_SHADER_STAGE_MISS_BIT_KHR;
 	}
 	else if (ext == "rchit")
 	{
-		return VK_SHADER_STAGE_CLOSEST_HIT_BIT_NV;
+		return VK_SHADER_STAGE_CLOSEST_HIT_BIT_KHR;
+	}
+	else if (ext == "rcall")
+	{
+		return VK_SHADER_STAGE_CALLABLE_BIT_KHR;
 	}
 
```

Comment:
For completion, there should be the support for "rahit"  -> VK_SHADER_STAGE_INTERSECTION_BIT_KHR

Reply:
Good point. I have added all missing ray tracing shader stages to the PR, including any hit and intersection stages.